traffic_ops/client

```diff
@@ -1 +1 @@
-v3-client
\ No newline at end of file
+v4-client
```

Comment:
We can actually just remove this symlink now. Its use was deprecated in the last release.